docs/trace/getting-started/README.md

```diff
@@ -74,12 +79,10 @@ An OpenTelemetry
 [TracerProvider](#tracerprovider)
 is configured to subscribe to the activities from the source
 `MyCompany.MyProduct.MyLibrary`, and export it to `ConsoleExporter`.
-`ConsoleExporter` simply displays it on the console. The `AlwaysOnSampler` is
-used by default which does not sample any traces.
```

Comment:
+1, btw I feel that "does not sample any traces" might be interpreted in very different ways depending on the background of the reader, the spec in general uses "create" and "record" as the verb rather than "sample".

Reply:
yup.